packages/react-spinner/src/components/Spinner/Spinner.test.tsx

```diff
@@ -19,10 +19,25 @@ describe('Spinner', () => {
     disabledTests: ['component-has-static-classname', 'component-has-static-classname-exported'],
   });
 
-  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
-
   it('renders a default state', () => {
     const result = render(<Spinner>Default Spinner</Spinner>);
     expect(result.container).toMatchSnapshot();
   });
+
+  it('renders Spinner with a label', () => {
+    const result = render(<Spinner label="Loading" />);
+    expect(result.container).toMatchSnapshot();
```

Comment:
I'd like to eliminate this snapshot test and change it to:

1. Assert the label element exists (something like `getByText("loading")`)
2. Assert the `role="progressbar"` SVG is present.